### PR TITLE
File permissions for build script, Reference not set error

### PR DESCRIPTION
**Pull request by Slesa**
* set the buld.sh scripts with permission to execute
* hint in the readme that sometimes the path to the nupk is needed for installation

### CODE REVIEW

**Comment by forki** [comment]
@theimowski please release. Thanks

**Comment by forki** [comment]
@slesa thanks for contributing

File: README.md [hunk 1]
```diff
@@ -27,6 +27,10 @@ The template does not include any Azure / other Cloud integration. Refer to [SAF
   * Runs `dotnet run --project src/Server/Server.fsproj` in root directory (note: Suave is launched on port **8085**)
   * Opens browser with url to Webpack development server (5 second delay after running client)
 
+* In case that dotnet -i fails with an 'Reference not set' error on Linux, try
+  * Uninstall a previous version: `dotnet new -u SAFE.template`
+  * Install the new template with its path: `dotnet new -i ./nupkg/SAFE.Template.<<version>>.nupkg
```
Comment (theimowski) [inline README.md]: Can you elaborate more on the 'Reference not set' issue? Should this mention rather go to the Contribution Guidelines? How do template users access nupkg directly?

**Comment by theimowski** [comment]
Thanks, released in 0.0.6.

**Pull request merged**